.eslintrc.json

```diff
@@ -4,10 +4,11 @@
         "es2021": true
     },
     "extends": [
-        "plugin:react/recommended",
         "eslint:recommended",
         "airbnb",
-        "plugin:@typescript-eslint/recommended"
+        "plugin:@typescript-eslint/recommended",
+        "plugin:react/recommended",
+        "prettier"
```

Comment:
I've added prettier here as well. Its a nice formatter and plays quite nicely with eslint. Shouldn't impact anyone else if they aren't using it.